app/scripts/modules/amazon/src/serverGroup/configure/wizard/AmazonCloneServerGroupModal.tsx

```diff
@@ -149,6 +160,7 @@ export class AmazonCloneServerGroupModal extends React.Component<
   }
 
   private submit = (command: IAmazonServerGroupCommand): void => {
+    this.normalizeCommand(command);
```

Comment:
I went back and forth for a minute on putting this in `convertServerGroupCommandToDeployConfiguration`, but IMO empty keys are ephemeral component/view state that should be scrubbed before they hit any transformer/writer code. Happy to be convinced otherwise.